Assets/Scripts/Battlefield/CreatureScripts/UnitStats.cs

```diff
@@ -9,12 +9,14 @@ public class UnitStats : MonoBehaviour
         [Header("Player Stats")]
         public int health;
         public int attack;
-        public int specialAttack;
+        public int magicAttack;
         public int defense;
-        public int specialDefense;
+        public int magicDefense;
         public int speedIntit;
         public int movement;
-
+        public int accuracy;
+        public int evasion;
+        public string role;
```

Comment:
This file should be removed. We already have an interface and class for this and this was not part of these issues tasking. I am willing to accept this PR with these changes as long as there is a high priority issue created for next timebox focused on removing this

Reply:
Will be fixed in #93